ops/terraform/services/pipeline/main.tf

```diff
@@ -21,10 +21,14 @@ locals {
   }
 
   # NOTE: nonsensitive service-oriented and common config
-  nonsensitive_common_map     = zipmap(data.aws_ssm_parameters_by_path.nonsensitive_common.names, nonsensitive(data.aws_ssm_parameters_by_path.nonsensitive_common.values))
-  nonsensitive_common_config  = { for key, value in local.nonsensitive_common_map : split("/", key)[5] => value }
-  nonsensitive_service_map    = zipmap(data.aws_ssm_parameters_by_path.nonsensitive.names, nonsensitive(data.aws_ssm_parameters_by_path.nonsensitive.values))
-  nonsensitive_service_config = { for key, value in local.nonsensitive_service_map : split("/", key)[5] => value }
+  nonsensitive_common_map    = zipmap(data.aws_ssm_parameters_by_path.nonsensitive_common.names, nonsensitive(data.aws_ssm_parameters_by_path.nonsensitive_common.values))
+  nonsensitive_common_config = { for key, value in local.nonsensitive_common_map : split("/", key)[5] => value }
+
+  nonsensitive_ccw_service_map    = zipmap(data.aws_ssm_parameters_by_path.nonsensitive_ccw.names, nonsensitive(data.aws_ssm_parameters_by_path.nonsensitive_ccw.values))
+  nonsensitive_ccw_service_config = { for key, value in local.nonsensitive_ccw_service_map : split("/", key)[6] => value }
+
+  nonsensitive_rda_service_map    = zipmap(data.aws_ssm_parameters_by_path.nonsensitive_rda.names, nonsensitive(data.aws_ssm_parameters_by_path.nonsensitive_rda.values))
+  nonsensitive_rda_service_config = { for key, value in local.nonsensitive_rda_service_map : split("/", key)[6] => value }
```

Comment:
So we're using the datasource for the splitting nomenclature? Just double checking so we can all be in agreement and consistently name things going forward. (I've got no problems with it myself)

Reply:
> So we're using the datasource for the splitting nomenclature? Just double checking so we can all be in agreement and consistently name things going forward. (I've got no problems with it myself)

The names of the data sources are informed by the specific hierarchies here. I think this is consistent with what we're doing elsewhere in the terraform. I'll admit that the recent terraservices are wanting for some additional standards, though. I agree!